app/src/main/java/org/kiwix/kiwixmobile/downloader/DownloadService.java

```diff
@@ -118,10 +118,12 @@ public int onStartCommand(Intent intent, int flags, int startId) {
     }
     if (intent.hasExtra(NOTIFICATION_ID) && (intent.getAction().equals(ACTION_PAUSE))) {
       if (KiwixMobileActivity.wifiOnly && !NetworkUtils.isWiFi(getApplicationContext())) {
+        Log.i("kiwixdownloadservice", "Not connected to WiFi, and wifiOnly is enabled");
```

Comment:
Please replace the repeated text with a single const value that's used throughout these Log messages. Also they may be more readable if you use camel case for the name...